lib/committee/schema_validator/option.rb

```diff
@@ -35,6 +36,7 @@ def initialize(options, schema, schema_type)
         @check_header        = options.fetch(:check_header, true)
         @coerce_recursive    = options.fetch(:coerce_recursive, true)
         @optimistic_json     = options.fetch(:optimistic_json, false)
+        @parse_response_by_content_type = options.fetch(:parse_response_by_content_type, false)
```

Comment:
I think this option should be true but we can't set for backward compatibility.
So when we upgrade major version, we should change default value.

But the user doesn't set option will get error.
So when the user doesn't set value, we should display a warning message like `please set parse_response_by_content_type = false because we'll change default value in next major version` to make the upgrade easier.

Please use `Committee.warn_deprecated` to display warning message
https://github.com/interagent/committee/blob/10734081293b7df808ded7e9a06a9bf97198a67b/lib/committee/middleware/base.rb#L79

Reply:
- parse_response_by_content_type = false
  - old behavior
- parse_response_by_content_type  = true
  - new behavior
- parse_response_by_content_type  = nil
  - show warning message and old behavior